DXMainClient/Startup.cs

```diff
@@ -191,14 +191,14 @@ private static void GenerateOnlineId()
                 string sid = new SecurityIdentifier((byte[])new DirectoryEntry(string.Format("WinNT://{0},Computer", Environment.MachineName)).Children.Cast<DirectoryEntry>().First().InvokeGet("objectSID"), 0).AccountDomainSid.Value;
 
                 Connection.SetId(cpuid + mbid + sid);
-                Registry.CurrentUser.CreateSubKey("SOFTWARE\\" + ClientConfiguration.Instance.InstallationPathRegKey).SetValue("Ident", cpuid + mbid + sid);
+                Registry.CurrentUser.CreateSubKey("SOFTWARE/" + ClientConfiguration.Instance.InstallationPathRegKey).SetValue("Ident", cpuid + mbid + sid);
             }
             catch (Exception)
             {
                 Random rn = new Random();
 
                 RegistryKey key;
-                key = Registry.CurrentUser.CreateSubKey("SOFTWARE\\" + ClientConfiguration.Instance.InstallationPathRegKey);
+                key = Registry.CurrentUser.CreateSubKey("SOFTWARE/" + ClientConfiguration.Instance.InstallationPathRegKey);
```

Comment:
Using forward slashes supposedly doesn't work in Windows registry paths.

Reply:
Resolved in a42d749.